dipy/segment/tests/test_metric.py

```diff
@@ -153,7 +153,7 @@ def dist(self, v1, v2):
 
             v1_normed = v1.astype(np.float64) / norm(v1.astype(np.float64))
             v2_normed = v2.astype(np.float64) / norm(v2.astype(np.float64))
-            cos_theta = np.dot(v1_normed, v2_normed.T)
+            cos_theta = np.dot(v1_normed, v2_normed.T).item()
```

Comment:
It happens here that the `v1` and `v2` values that the `dist` method receives are 1x3 numpy arrayz (`[[]]`) instead of full the full 18x3 matrix in the caller. I do not understand that behavior. The other solution would be to simply squeeze `v1` and `v2` in the `dist` method, but I ignore the consequences of that.